### PR TITLE
chore: improve key rotation api docs

### LINKED ISSUE #7996: The Key Rotations API docs review
### Issue

The following information in the docs states “for a given party” but the field is called `nodeID` and is also optional:

https://docs.vega.xyz/testnet/api/rest/data-v2/trading-data-service-list-key-rotations
https://docs.vega.xyz/testnet/api/rest/data-v2/trading-data-service-list-ethereum-key-rotations

The key rotation API docs needs to be reviewed and cleaned up to make accurate.

### PR DESCRIPTION
**Pull request by wwestgarth**
closes #7996

### CODE REVIEW

File: protos/sources/vega/events/v1/events.proto [hunk 1]
```diff
@@ -506,27 +506,27 @@ message ValidatorRankingEvent {
...
+// An event that contains information about a Vega key rotation
 message KeyRotation {
-  // Node id of the node
+  // Identifier of the node that rotated their Vega key
   string node_id = 1;
-  // Old Vega public key of validator node
+  // Vega public key that was previously associated with the node
   string old_pub_key = 2;
-  // New Vega public key of validator node
+  // Vega public key that was next associated with the node
```
Comment (candida-d) [inline protos/sources/vega/events/v1/events.proto:515]:
```suggestion
  // Vega public key that is newly associated with the node
```
perhaps?

File: protos/sources/vega/events/v1/events.proto [hunk 2]
```diff
@@ -506,27 +506,27 @@ message ValidatorRankingEvent {
...
   string new_pub_key = 3;
-  // Height of the block where key rotation took effect
+  // Block height where the key rotation took effect
```
Comment (candida-d) [inline protos/sources/vega/events/v1/events.proto:517]:
```suggestion
  // Block height when the key rotation took effect
```

File: protos/sources/vega/events/v1/events.proto [hunk 3]
```diff
@@ -506,27 +506,27 @@ message ValidatorRankingEvent {
...
   uint64 block_height = 4;
 }
 
-// An Ethereum key rotation events contains information about Ethereum key rotation
+// An event that contains information about an Ethereum key rotation
```
Comment (candida-d) [inline protos/sources/vega/events/v1/events.proto:521]:
```suggestion
// Event that contains information about an Ethereum key rotation
```

File: protos/sources/vega/events/v1/events.proto [hunk 4]
```diff
@@ -506,27 +506,27 @@ message ValidatorRankingEvent {
...
+  // Identifier of the node that rotated their Ethereum key
   string node_id = 1;
-  // Old Ethereum address of validator node
+  // Ethereum address that was previously associated with the node
   string old_address= 2;
-  // New Ethereum address of validator node
+  // Ethereum address that was next associated with the node
   string new_address = 3;
-  // Height of the block where key rotation took effect
+  // Block height where the key rotation took effect
```
Comment (candida-d) [inline protos/sources/vega/events/v1/events.proto:529]:
```suggestion
  // Block height when the key rotation took effect
```

**Review by jeremyletang — APPROVED** [review]

**Review by candida-d — APPROVED** [review]

**Pull request merged**